framework/include/loops/ThreadedElementLoop.h

```diff
@@ -55,6 +55,22 @@ class ThreadedElementLoop : public ThreadedElementLoopBase<RangeType>
 
 protected:
   FEProblemBase & _fe_problem;
+
+  /**
+   * Routine to output the ordering of objects within a vector. These objects must implement the
+   * name() routine, and it must return a string or compatible type.
+   *
+   * @tparam RangeType the type of the element-based loop
```

Comment:
```suggestion
```
That template parameter is for the class and so should be in the class's doxygen